### DISCUSSION TITLE
Never ending process on Mac

### DISCUSSION THREAD

**Discussion by iPlumb3r** [lightningrodlabs/acorn#279] [Q&A]
Hi Team, 

I just downloaded the Acorn 4.0.0 version here : https://github.com/lightningrodlabs/acorn/releases/tag/v4.0.0-alpha
Version for mac Ventura 13.3.1

When trying to install the prompt propose me to install a more fresh version (5.0.2)

But after clicking on the "Update Now" button, I got a never ending "Preparing your update" screen

Best regards

Bernard

**Comment by Connoropolous** [comment]
Hi Bernard,

If it's not a problem for you, can you just go here and download the newest version manually?

https://github.com/lightningrodlabs/acorn/releases/tag/v5.0.2-alpha

**Comment by iPlumb3r** [comment]
Hi Connor, 

I encountered the very same problem when chosing "Update Now"
<img width="1188" alt="Capture d’écran 2023-05-04 à 10 58 33" src="https://user-images.githubusercontent.com/46555634/236158206-5ca90eda-a1a0-47c8-b7fd-b84862723cfd.png">

Looks like it worked if I choose "I'll update later
